src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin.cxx

```diff
@@ -338,21 +557,69 @@ add_subset_sensitivity(TargetT& sensitivity, const int subset_num) const
     const int min_segment_num = proj_data_info_sptr->get_min_segment_num();
     const int max_segment_num = proj_data_info_sptr->get_max_segment_num();
 
+#ifdef STIR_TOF
+    int min_timing_pos_num = use_tofsens ? this->proj_data_info_sptr->get_min_tof_pos_num() : 0;
+    int max_timing_pos_num = use_tofsens ? this->proj_data_info_sptr->get_max_tof_pos_num() : 0;
+#endif
+
     this->projector_pair_sptr->get_back_projector_sptr()->
       start_accumulating_in_new_target();
 
     // warning: has to be same as subset scheme used as in distributable_computation
+#ifdef STIR_OPENMP
+#ifdef _WIN32
+    #pragma omp parallel for
+#else
+    #pragma omp parallel for collapse(2) schedule(dynamic)
+#endif
+#endif
     for (int segment_num = min_segment_num; segment_num <= max_segment_num; ++segment_num)
     {
       for (int view = proj_data_info_sptr->get_min_view_num() + subset_num;
           view <= proj_data_info_sptr->get_max_view_num();
           view += this->num_subsets)
       {
-        const ViewSegmentNumbers view_segment_num(view, segment_num);
+          const ViewSegmentNumbers view_segment_num(view, segment_num);
 
-        if (! this->projector_pair_sptr->get_symmetries_used()->is_basic(view_segment_num))
-          continue;
-        this->add_view_seg_to_sensitivity(view_segment_num);
+          if (! this->projector_pair_sptr->get_symmetries_used()->is_basic(view_segment_num))
+            continue;
+          //        this->add_view_seg_to_sensitivity(view_segment_num);
```

Comment:
why was the code moved here from this function? If possible I'd prefer to have it in the same place (will also generate less conflicts when merging to TOF)

Reply:
ah well. fun to come then